anonymising_data/retrieve_data/retrieve_data.py

```diff
@@ -13,8 +14,16 @@ class RetrieveData:
     Class to retrieve data.
     """
 
-    def __init__(self, config):
+    def __init__(self, config, concepts=None, person_id=None):
+        if person_id is None:
+            person_id = []
+        if concepts is None:
+            concepts = []
         self._query_file = config.output_query_file
+        if config.testing and config.cpet:
+            q = Query(config, concepts, create_link_query=False, person_id=person_id)

```

Comment:
When it is cpet sql, we called Query twice. One in anonymise, now here in retrieve.